astrolabe/cli.py

```diff
@@ -613,8 +615,11 @@ def delete_test_cluster(ctx, spec_test_file, workload_file, org_id, project_name
     if project:
         try:
             ctx.obj.client.groups[project.id].clusters[cluster_name].delete()
-        except AtlasApiBaseError:
-            pass
+            print(f"{msg} done.")
+        except AtlasApiBaseError as e:
+            pprint(e)
+    else:
+        print(f"Project {project_name} not found!")
```

Comment:
Should we raise an exception here if we attempted to delete a cluster but couldn't find the project?

Reply:
Same answer as above